packages/gatsby-source-wordpress/style.css

```diff
@@ -0,0 +1,4 @@
+.inline-gatsby-image-wrapper,
```

Comment:
Do we have to include it globally? Could we inject a style tag only when necessary?

Reply:
We do have to keep it global unfortunately. The html transformation happens on all fields at any depth in processed nodes - so we don't really know when someone has used an html field with Gatsby images in it. We could export this css file instead of including it and have people import it themselves but it will probably break current users layouts if we do that.

Reply:
It's fine, it was more a question. It's not a lot of css.